front/app/containers/App/index.tsx

```diff
@@ -172,7 +172,7 @@ class App extends PureComponent<Props, State> {
       navbarRef: null,
       mobileNavbarRef: null,
       locale: null,
-      invitationDeclined: false,
+      signUpInModalClosed: false,
```

Comment:
I added `signUpInModalOpened: boolean;` in my earlier PR, it's a few lines above this one. I think it does the same thing so we can probably just keep one of them. Also applies to the setState function passed down into the modal I believe

Reply:
But I need to pass `false` to `openSignUpInModalIfNecessary` when the modal was closed (so, we don't need to open it again). And your hook runs when the `SignUpInModal` is rendered.

Could you maybe show how it can be implemented to make it clearer for me?

Reply:
Yeah let's pair on it tomorrow, maybe I misunderstood 👍